src/docs/product/performance/performance-at-scale/index.mdx

```diff
@@ -100,6 +100,12 @@ Retention priorities and your SDK sampling configuration act independently of ea
 
 To send 100% of your transaction events, set your [tracesSampleRate](/platform-redirect/?next=/performance/) in the SDK to 1.0. If you want to offset overhead, (with high-volume back-end applications, for example), set a lower value, or switch to sampling selectively by using the <PlatformIdentifier name="traces-sampler" /> to filter your transactions based on contextual data.
 
+<Note>
+
+Please note that Performance at Scale doesn't automatically mange your quota. If you increase your SDK sampling rate, you'll also need to increase the volume of transactions you send to Sentry. This may require that you adjust your [transactions quota](/product/accounts/quotas/manage-transaction-quota/).
```

Comment:
```suggestion
Please note that Performance at Scale doesn't automatically mange your quota. If you increase your SDK sampling rate, you'll also increase the volume of transactions you send to Sentry. This may require that you adjust your [transactions quota](/product/accounts/quotas/manage-transaction-quota/).
```